sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DeploymentStep.java

```diff
@@ -125,8 +141,7 @@ public synchronized void update(Protos.TaskStatus status) {
                     podInstanceRequirement.getPodInstance(),
                     CommonIdUtils.toTaskName(status.getTaskId()));
         } catch (TaskException e) {
-            logger.error(String.format("Failed to update status for step %s", getName()), e);
-            setStatus(super.getStatus()); // Log status
+            logger.error(String.format("Failed to update status for step %s, status: {} ", getName()), e, getStatus());
```

Comment:
this two-tier formatting is a bit wild. maybe just go with `... for step %s, status: %s", getName(), getStatus()), e);`

Reply:
oh, yes.  Didn't see this.